requirements.txt

```diff
@@ -0,0 +1 @@
+pyvista
```

Comment:
no need for this if using `ghcr.io/pyvista/pyvista:latest`. PyVista comes pre-installed in that Docker image. Though it's good to be explicit

Reply:
```suggestion
pyvista==0.33.2
```